### ISSUE TITLE
Tutorial request

### ISSUE THREAD

**Issue by timj** [lsst-dm-tutorial/lsst2017#1]
Please allow me to participate in this tutorial.

**Comment by dkirkby** [comment]
The README instructions say to:

> request membership in the lsst-dm-tutorial GitHub organization

but I only see one member @TallJimbo and no way to request membership.

**Comment by TallJimbo** [comment]
@timj, I think you should already have access by virtue of being in the `lsst` org.

@dkirkby, I've sent you an invite. There are other owners of the org, but making membership public is an opt-in thing that no one else has done.

Anyone else who wants an account and sees this issue should just reply here.  There doesn't seem to be a button for requesting membership.

**Comment by timj** [comment]
@TallJimbo Yes. Mine was a test requested by @frossie and she should have closed it.

**Comment by drphilmarshall** [comment]
Terribly late, but can I request membership so I can check out the tutorial please? Thanks!
